pbs/pbsrequest.go

```diff
@@ -30,20 +37,42 @@ type Bids struct {
 	Params     json.RawMessage `json:"params"`
 }
 
+type PBSVideo struct {
```

Comment:
I definitely can add docs here. But it will be the only documented structure then. Will it be OK?